### PR TITLE
fix: path problems, add doc GitHub Page Actions

### PR DESCRIPTION
**Pull request by yihong0618**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -39,3 +39,11 @@ commands =
     black . --check --diff
     isort --profile black . -c --diff
     pyright
+
+[testenv:docs]
+basepython = python3.9
+deps = {[base]deps}
+skip_install = true
+commands =
+    pip install .
```
Comment (beeender) [inline tox.ini:49]: not needed

File: requirements-dev.txt [hunk 2]
```diff
@@ -3,3 +3,6 @@ isort==5.10.1
 pytest==7.0.1
 pglast==3.10
 pyright==1.1.250
+psycopg2==2.9.3
```
Comment (beeender) [inline requirements-dev.txt]: not needed, it is in the `requirments.txt` already.

File: doc/README.md [hunk 3]
```diff
@@ -1,2 +1,8 @@
-Require `Sphinx`  
-Use `make builder` where `builer` can be `HTML`, `Latex`, ... 
\ No newline at end of file
+## Require `Sphinx`
```
Comment (beeender) [inline doc/README.md:1]: remove this `README.md` and add a section in the root's readme about how to use tox to generate doc.

And remove the makefile in this dir.

File: .github/workflows/doc.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - name: Install dependencies
+      # pip install . --> for doc generate
+      run:  |
+        python -m pip install --upgrade pip
+        pip install -r requirements-dev.txt
+        pip install .
+    - name: Make doc
+      run: |
+         sphinx-build -M html doc/source build 
+         mkdir -p public
```
Comment (beeender) [inline .github/workflows/doc.yml:47]: line 25-39 can be simply changed to `tox -e docs` I think?

Comment (yihong0618) [reply .github/workflows/doc.yml]: done

File: doc/README.md (hunk 3 above)
Comment (yihong0618) [reply doc/README.md]: done

**Review by beeender — APPROVED** [review]

File: greenplumpython/__init__.py [hunk 5]
```diff
@@ -1,8 +1,8 @@
-from .db import Database, database
-from .expr import get_type
-from .func import create_aggregate  # type: ignore
-from .func import create_array_function  # type: ignore
-from .func import create_function  # type: ignore
-from .func import aggregate, function
-from .op import operator
-from .table import Table, table, values
+from greenplumpython.db import Database, database
```
Comment (Quasimewdo) [inline greenplumpython/__init__.py:1]: I tried with this syntax, but had error, was it because of line 54 `html_static_path = ["_static"]` in conf.py?

Comment (yihong0618) [reply greenplumpython/__init__.py:1]: we need pip install . to install it as moudle

**Review by Quasimewdo — COMMENTED** [review]
Brilliant, thanks!

File: greenplumpython/__init__.py (hunk 5 above)
Comment (Quasimewdo) [reply greenplumpython/__init__.py:1]: Learnt a new thing

**Pull request merged**